### ISSUE TITLE
 Black text on gray background in HTML view of arXiv:2312.04263v1

### ISSUE THREAD

**Issue by mohammad-akhlaghi** [arXiv/html_feedback#130]
### Description

The text of https://browse.arxiv.org/html/2312.04263v1 is black on a gray background! This makes it particularly hard to read! I noticed in the HTML of other papers (for example https://browse.arxiv.org/html/2312.05358v1 ) that this does not happen (the text is white).

### (Required) What is this issue most closely related to? Select one.

Formula

### Internal issue ID

f8606df9-ee66-4a73-87c0-d24c19425af4

### Paper URL

https://browse.arxiv.org/html/2312.04263v1

### Browser

Firefox/120.0

### Device Type

Desktop

**Comment by dginev** [comment]
Interesting case.
The document source explicitly sets a black color for the main text:
```tex
%% Color management.
\usepackage{xcolor}
\color{black} % Color of main text.
```

Currently, latexml respects that too strictly, by setting a `style="color:#000000;"` attribute on all mainmatter elements in HTML. That is quite hard to override in dark mode -and worse- it is not really possible to tell apart which colors are meant as intentional highlight, and which ones are just precautionary defensive settings (as in this example).

So two directions for change:
1. Removing that `\color{black}` line from the source (if the author had the choice)
2. Improve latexml to both 1) avoid setting color explicitly if it matches the current setting, 2) use a `class` color vocabulary instead of `style` for the standard entries, making it easier to theme in dark mode with regular CSS.

**Comment by SBBCornell** [comment]
A temporary and quick suggestion for @mohammad-akhlaghi: turn off dark mode by clicking on the little sun icon in the upper right corner. That will make the HTML page legible for now.

**Comment by mohammad-akhlaghi** [comment]
You are right! When the dark-mode is off, the two papers are displayed similarly. My computer defaults to dark-mode.

The strange thing is that both these papers have been created with AASTeX, but one displays white fonts in dark mode, while the other displays black fonts in dark mode.

Thanks a lot for the fast feedback.

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved.

We have regenerated the article and you should be able to see an improvement.

https://arxiv.org/html/2312.04263v1
